src/components/InstallModalContent.js

```diff
@@ -1,36 +1,88 @@
-import * as React from 'react';
+import * as React from "react";
 
 export class InstallModalContent extends React.Component {
-
-  constructor (props) {
+  constructor(props) {
     super();
-    this.state = 
-      {value:'', copied:false};
-      
-    
+    this.state = { value: "", copied: false };
+  }
+
+  copyToClipboard = e => {
+    console.log("got it");
+    var textField = document.createElement("textarea");
+    textField.innerText = this.command();
+    document.body.appendChild(textField);
+    textField.select();
+    document.execCommand("copy");
+    textField.remove();
+    e.target.focus();
+    setTimeout(
+      function() {
+        this.setState({ copied: false });
+      }.bind(this),
+      5000
+    );
+    this.setState({ copied: true });
+  };
+
+  command() {
+    return `kubectl create -f ${this.props.expcrdurl}`;
   }
 
   render() {
     return (
       <div className="install-modal-content">
         <div className="modal-header">
-          <div className="modal-img-placeholder"> <img src={this.props.logo} /></div>
+          <div className="modal-img-placeholder">
+            {" "}
```

Comment:
what is this??

Reply:
I think this is something my formatter did. I'll revert this out